### PR TITLE
l10n.eclass: update l10n_find_plocales_changes()

### PR DESCRIPTION
**Pull request by rindeal**
This update does
- allow to span PLOCALES over multiple lines
- make code more readable

Gentoo-Bug: 513242

@yngwin @jlec

### CODE REVIEW

File: eclass/l10n.eclass [hunk 1]
```diff
@@ -77,18 +77,23 @@ l10n_for_each_disabled_locale_do() {
...
-	for x in ${2}*${3} ; do
-		x=${x#"${2}"}
-		x=${x%"${3}"}
-		current+="${x} "
+	local dir="${1}" pre="${2}" post="${3}"
+
+	einfo "Looking in '${dir}' for new locales ..."
+	pushd "${dir}" >/dev/null || die "Cannot access ${dir}"
+	local current=
+	for l in ${pre}*${post} ; do
```
Comment (jlec) [inline eclass/l10n.eclass]: ```l``` should be local as well.

Comment (rindeal) [inline eclass/l10n.eclass]: Thank you very much @jlec , didn't know bash doesn't limit scope of loop vars.

**Comment by rindeal** [comment]
What's holding this back?

**Comment by monsieurp** [comment]
@rindeal: Hi. Eclass changes must be discussed on the gentoo-dev mailing first. Not all Gentoo devs use and/or like Github. Thank you.

**Comment by rindeal** [comment]
I announced it there on the same day as here, but no one responded and
apparently no one read it.

**Comment by rindeal** [comment]
https://github.com/gentoo/gentoo/commit/becf07543e1dcc97fc5c3d099c0d393f77997b32

**Comment by rindeal** [comment]
I guess this can be closed now.

For anyone interested, I've completely reworked the eclass and added many cool features, while keeping full backwards compatibility. It can be seen in action in my overlay - https://github.com/rindeal/gentoo-overlay/blob/master/eclass/l10n.eclass

**Pull request closed without merging**